charts/weave-gitops/templates/admin.yaml

```diff
@@ -0,0 +1,49 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: wego-admin-read-resources
+  namespace: {{.Values.namespace}}
+subjects:
+  - kind: User
+    name: wego-admin
```

Comment:
So. I think no, and I think that's not my fault.

As far as I can tell, every time you log in using a (local?) user, your will be logged in as the same user, and it's this one: https://github.com/weaveworks/weave-gitops/blob/main/pkg/server/auth/token_signer_verifier.go#L54 - I've changed it from "admin" to "wego-admin" because v2 swapped it for a constant with a slightly different value, but now that I think deeper about it I'm not sure a constant user makes all that much sense.

Reply:
yep i tried changing this and it did not work. the last missing piece for the multi-user bit

from what i understand what we want is:
- I can create a new k8s user of any name with its own role + binding
- I can create a new secret with any name with the username being the same name as the role above
- I can log in as that user

This may affect how the login works, as now it seems we look up very specific secret and user names. We may need to do something with labels. I will create a ticket.

Reply:
Ah yes, so currently the signed token uses `wego-admin`, it doesn't use the username. This is somewhat confusing but it should work if you use `wego-admin` in RBAC manifests. The username is just used during sign in but not anywhere else yet. We should probably change that (in a follow-up PR) for our own sanity.